crowbar_framework/app/models/service_object.rb

```diff
@@ -277,7 +287,7 @@ def restore_to_ready(nodes)
 #   dequeue_proposal - remove item from queue and clean up
 #   process_queue - see what we can execute
 #
-  def queue_proposal(inst, elements, deps, bc = @bc_name)
+  def queue_proposal(inst, element_order, elements, deps, bc = @bc_name)
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for queue_proposal is too high. [56.67/25]